state/migration_import_test.go

```diff
@@ -1309,31 +1309,33 @@ func (s *MigrationImportSuite) TestSpaces(c *gc.C) {
 
 func (s *MigrationImportSuite) TestFirewallRules(c *gc.C) {
```

Comment:
Is the test about explicit overwriting of existing rules? 
If yes I would rename test method imo.
`TestOverwriteFirewallRules` or something along the line, wdyt?

Reply:
When migrating from one model to another, I want to remove the old firewall rule during an export and import, to ensure that it's not the old rule that I'm picking up. So when I'm testing in the new state, it's in no way the old rule.

Reply:
The test itself makes a lot of sense to me. I was thinking about renaming the test name as one could argue that the name is too generic.
IMO your above description would great as a method name/method description.